app/src/main/java/com/jerboa/model/PersonProfileViewModel.kt

```diff
@@ -268,6 +269,21 @@ class PersonProfileViewModel(personArg: Either<PersonId, String>, savedMode: Boo
         }
     }
 
+    fun distinguishComment(form: DistinguishComment) {
+        viewModelScope.launch {
+            distinguishCommentRes = ApiState.Loading
+            distinguishCommentRes = API.getInstance().distinguishComment(form).toApiState()
+
+            when (val distinguishRes = distinguishCommentRes) {
+                is ApiState.Success -> {
+                    updateComment(distinguishRes.data.comment_view)
+                }
+
+                else -> {}
```

Comment:
Display failure as toast?

Reply:
Check all the ones above this. If we do that we can do it for all of them as part of a cleanup later.